# AudioStamped.msg
std_msgs/Header header  # Standard ROS header with timestamp
uint8[] data            # Raw audio data
